apps/sp-student/src/pages/assignment/Assignment.js

```diff
@@ -0,0 +1,73 @@
+import React, { useMemo, useState } from 'react';
+import PDFViewer from 'pdf-viewer-reactjs';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Select,
+} from '../../../../../libs/ui-shared/src/lib/components';
+import Pdf from '../../../../../libs/ui-shared/public/sample.pdf';
```

Comment:
Suggest Handling pdf import in separate file which takes week and assignment ID as props.

Reply:
Noted your observation, write now it is to get the UI correct. We will take care of it while writing the logic.